web/src/beta/lib/core/engines/Cesium/useEngineRef.ts

```diff
@@ -468,6 +474,19 @@ export default function useEngineRef(
         const assignments = getOverriddenScreenSpaceCameraOptions(options);
         Object.assign(controller, assignments);
       },
+
+      keyboardCameraRotate: () => {
```

Comment:
The word `keyboard` seems meanning less here. I guess it's from the behavior of View 3.0, but as a API of reearth it should be something like `rollCameraHorizontal`

Reply:
We can do this just by using `reearth.camera.setView` in plugin side. So I think we don't need this.

Reply:
@keiya01 If we use reearth.camera.setView for the plugin we will have to pass the camera position which delays as @airslice mentioned last time and make the camera not smooth

Reply:
I think `reearth.camera.setView` doesn't update some state in ReEarth. It just update the Cesium's camera:eyes: